.github/ISSUE_TEMPLATE/failing_test_jenkins_template.yaml

```diff
@@ -0,0 +1,115 @@
+title: 'CI: '
+labels: ["area/CI", "ci/flake"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        - Set title to be in the format `CI: <test-name>`.
+        - Copy-paste the output the test failure.
+        - Upload the zip file generated from that test failure.
+        - Copy-paste the link of the CI build where that test failure has happened.
+        - Include any output from logs that you think may be relevant (to ease GitHub searches).
+  - type: textarea
+    id: test-name
+    attributes:
+      label: Test Name
+      description: Test suite and name of the test which failed
+      placeholder: K8sFQDNTest Validate that FQDN policy continues to work after being updated
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: failure-output
+    attributes:
+      label: Failure Output
+      description: Failure message for the test.
+      placeholder: |
+        FAIL: Can't connect to to a valid target when it should work
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: stack-trace
+    attributes:
+      label: Stack Trace
+      description: Stack trace for the failure, as reported by Jenkins. This will be automatically formatted into code, so no need for backticks.
+      placeholder: |
+        /home/jenkins/workspace/Cilium-PR-K8s-1.22-kernel-4.19/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:527
+        Can't connect to to a valid target when it should work
+        Expected command: kubectl exec -n default app2-58757b7dd5-pvvvw -- curl --path-as-is -s -D /dev/stderr --fail --connect-timeout 5 --max-time 20 --retry 5 http://vagrant-cache.ci.cilium.io -w "time-> DNS: '%{time_namelookup}(%{remote_ip})', Connect: '%{time_connect}',Transfer '%{time_starttransfer}', total '%{time_total}'"
+        To succeed, but it failed:
+        Exitcode: 28
+        Err: exit status 28
+        Stdout:
+             time-> DNS: '0.000016()', Connect: '0.000000',Transfer '0.000000', total '5.000836'
+        Stderr:
+             command terminated with exit code 28
+      render: shell
+    validations:
+      required: true
+  - type: textarea
+    id: stack-output
+    attributes:
+      label: Standard Output
+      description: Standard output messages for the failure, as reported by Jenkins. This will be automatically formatted into code, so no need for backticks.
+      placeholder: |
+        Cilium pods: [cilium-9gp4w cilium-vpsn6]
+        Netpols loaded:
+        CiliumNetworkPolicies loaded: default::fqdn-proxy-policy.yaml
+        Endpoint Policy Enforcement:
+        Pod   Ingress   Egress
+        Cilium agent 'cilium-9gp4w': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 37 Failed 0
+        Cilium agent 'cilium-vpsn6': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 33 Failed 0
+      render: shell
```

Comment:
I wonder if there is a renderer available for displayed in `<details><summary>...</summary> ... </details>` blocks?

Reply:
I wondered too, but I haven't been able to find a list of the existing renderers :/. I could look further, or reach out to support.
But even then, we'd need a renderer that does _both_ the `<details>` block and the code formatting.